File: Actividad.cpp
#include <iostream>
#include <string>
using namespace std;

int main (){
  
  float num1;
  float num2;
  string Operador;

  cout <<"Ingrese el primer numero: ";
  cin >> num1;

  cout <<"\n Ingrese el segundo numero: ";
  cin >> num2;

  cout <<"\n Ingrese el Operador (+, -, /, *): ";
  cin >> Operador;

  if (Operador == "+") {
    cout << "\n El resultado de la suma es: " << (num1 + num2) << endl ;
  }else if (Operador == "-"){
    cout << "\n El resultado de la resta es: "<< (num1 - num2) << endl ;
  }else if (Operador == "/"){
    cout << "\n El resultado de la division es: "<< (num1 / num2) << endl ;
  }else if (Operador == "*"){
    cout << "\n El resultado de la multiplicacion es: "<< (num1 * num2) << endl ;
  }else{
    cout << "\nOperador incorrecto";
  }

} 

File: Actividad.js
let num1 = 4;
let num2 = 3;
let Operador =  prompt ("Ingrese el operador: ").toLocaleLowerCase();


if (Operador === "suma") {
  console.log(num1 + num2)
}else if (Operador === "resta") {
  console.log(num1 - num2)  

}else if (Operador === "multiplicar") {
  console.log(num1 * num2)
  
}else if (Operador === "dividir") {
  console.log(num1 / num2)
  
}else {
  console.log("Opción no validad")
}